### DISCUSSION TITLE
jbangreleaser?

### DISCUSSION THREAD

**Discussion by maxandersen** [jbangdev/jbang#362] [General]
While working on snap Distro for jbang I bumped into goreleaser.com - a library(?) to go build, package and release go apps via homebrew, scoop and snap making it quite easy to build and release go cli apps. 

I realized we actually have these pieces in jbang too - just not packaged/gathered nicely. 

If anyone interested in exploring this and make something happen let me know as i don’t have enough hours in the day to make this happen right now but would love to see it happen. 

Wdyt? Interested in seeing this happen ?

**Comment by jeffmaury** [comment]
Will have a look at goreleaser and may help
